NodeJS/src/RestClient.ts

```diff
@@ -388,6 +391,27 @@ export class RestClient {
       return response.data
     })
   }
+
+  public async getAddressBook() {
+    const subAccountAppId = await this.getSubAccountAppId()
+    const url = `${this.baseUrl}/${ADDRESS_BOOK_PATH}/${subAccountAppId}`
+    const options = this.prepareGetOptions(url)
+    return axios(options).then((response: { data: any }) => {
+      return response.data
+    })
+  }
+
```

Comment:
Do we need to expose this endpoint?

Reply:
according to https://spokencloud.atlassian.net/browse/ENG-19798: ContactService SDK: Add ContactService endpoints to interface.

Reply:
My understanding is we will expose the search endpoint only. But maybe @justindstein  has some different idea?

Reply:
related to your security concern. in SDK accessible subAccountAppId is fetched from currentUser, so SDK users will not be able to get Address Books from improper Sub-accounts.

Reply:
Thanks. It really help to make the API easier to be used. Regarding the security issue, we all know that the SDK cannot prevent users from changing the code on browser, so the issue is still there.

Reply:
agree about browser issue.